apps/generator-canonical-ds/package.json

```diff
@@ -28,14 +28,15 @@
     "check:ts": "tsc --noEmit"
   },
   "dependencies": {
-    "yeoman-generator": "5.9.0"
+    "yeoman-generator": "7.3.3"
   },
   "devDependencies": {
     "@biomejs/biome": "^1.9.4",
-    "@canonical/biome-config": "^0.2.1-experimental.0",
-    "@canonical/typescript-config-base": "^0.2.1-experimental.0",
-    "@types/node": "^22.9.1",
-    "@types/yeoman-generator": "5.2.14",
+    "@canonical/biome-config": "^0.3.1-experimental.0",
+    "@canonical/typescript-config-base": "^0.3.1-experimental.0",
+    "@types/debug": "^4.1.12",
+    "@types/node": "^22.10.1",
+    "@types/json-schema": "^7.0.15",
```

Comment:
`debug` and `json-schema` types are now needed when compiling `yeoman-generator` with Typescript.